feat: init policy template (#2)
gravitee-io/gravitee-policy-template: README.adoc

== Description

A policy template to fork and use as a quick starter.

> This policy will compare `X-Template-Policy` header value with its configuration `errorKey` field, if both values are equal, then the policy will make the request or response fail.

Reviewer: ```suggestion
This policy will compare `X-Template-Policy` header value with its configuration `errorKey` field, if both values are equal, then the policy will interrupt the request with a failure execution error.
```
